inc/shortcodes/attributions/class-attachments.php

```diff
@@ -246,7 +246,7 @@ function attributionsContent( $attributions ) {
 						sprintf(
 							'%1$s %2$s %3$s',
 							$license_prefix,
-							( isset( $attribution['license'] ) ) ?
+							( isset( $attribution['license'] ) && isset( $supported[ $attribution['license'] ] ) ) ?
```

Comment:
This is unrelated to this issue, but fixes a warning I was getting when loading admin pages with WP debug turned on.